recipes/hipace/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "hipace" %}
+{% set version = "23.03" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Hi-PACE/hipace/archive/v{{ version }}.tar.gz
+  sha256: f32104af1e7ca33b7ef8a3fd7964ee62db13a68a00af305d4ace5d0e86e2efb2
+
+build:
+  number: 0
+  run_exports:
+    # releases are not (yet) compatible
+    - {{ pin_subpackage('hipace', max_pin='x.x') }}
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - ccache        # [unix]
+    - clang         # [win]
+    - cmake >=3.20.0
+    - lld           # [win]
+    # TODO: add libxml2 dep in lld-feedstock
+    - libxml2       # [win]
+    - make          # [unix]
+    - ninja         # [win]
+    - libgomp       # [linux]
+    - llvm-openmp   # [osx or win]
+    - pkgconfig
+  host:
+    - fftw =*=nompi_*
+    - openpmd-api =*=nompi_*
+
+test:
+  requires:
+    - bash  # [not win]
+    - python
+    - numpy
+    - openpmd-api =*=nompi_*
+    - openpmd-viewer
+    - pandas
+    - pytest
+    - python
+    - scipy
+  source_files:
+    - examples/
+    - tests/
+  files:
+    - test.sh   # [unix]
+    - test.bat  # [win]
+  commands:
+    - test -f ${PREFIX}/bin/hipace.NOMPI.NOACC.DP  # [unix]
+    - if not exist %LIBRARY_PREFIX%\bin\hipace.NOMPI.NOACC.DP.exe exit 1  # [win]
+    # smoke tests: execute
+    - ./test.sh      # [unix and build_platform == target_platform]
+    - call test.bat  # [win]
```

Comment:
For each platform, please add an existence test for a header file, and a cmake file. Also test that the library is shared and not static. This will hint to readers of the recipe that these files are expected to exist in the package without them needing to download it.

Reply:
The goal is only to install a binary, which we check three lines above.

I disabled install of all other artifacts, thanks for catching!

Reply:
Update: now a clean
```
-- Install configuration: "Release"
-- Installing: $PREFIX/bin/hipace.NOMPI.NOACC.DP
```

Thanks for catching this!